googlemaps/client.py

```diff
@@ -272,6 +287,10 @@ def _generate_auth_url(self, path, params, accepts_clientid):
 from googlemaps.roads import snap_to_roads
 from googlemaps.roads import speed_limits
 from googlemaps.roads import snapped_speed_limits
+from googlemaps.places import places
+from googlemaps.places import place
+from googlemaps.places import photo
+from googlemaps.places import autocomplete
```

Comment:
I thought so too originally, and all most of the methods prefixed with "places_", but then when I looked at how the modules for all the other services were implemented, none of them really did that, so I changed it for consistency.